README.md

```diff
@@ -1 +1,181 @@
-# Hello_Microverse_Project
\ No newline at end of file
+<a name="Hello Microverse Project"></a>
+
+
+<div align="center">
+  
+
+  <h3><b>Hello Microverse Project</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Hello Microverse Project] <a name="about-project"></a>
+
+> A portfolio I built with html and css.
+
+
+## 🛠 Built With <a name="built-with"></a>
+1. html
+2. css
+3. Linters
+
+### Tech Stack <a name="tech-stack"></a>
+1. html
+2. css
+3. Linters
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://indexhtml.org/">HTML</a></li>
+    <li><a href="https://styles=css/">CSS</a></li>
+    <li><a href="https://linters/">Linters</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+
+Added index.html file
+Added style.css file
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Live-demo will br added soon.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
Since you do not have a deployed version of the project, I think you can feel free to remove the live demo section from your readme